src/pages/Search.tsx

```diff
@@ -1,9 +1,9 @@
-import { Form, useLoaderData, useSubmit } from 'react-router-dom'
+import { Form, useLoaderData, useSubmit, Link } from 'react-router-dom'
 import Agent from '../types/Agent'
 import SearchResult from '../components/SearchResult'
 import { useEffect } from 'react'
 import { debounce } from 'lodash'
-import { Button, Card } from 'react-bootstrap'
+import Card from 'react-bootstrap'
```

Comment:
You'll still need the destructuring `{}` for this to work.

```suggestion
import { Card } from 'react-bootstrap'
```

Reply:
React-bootstrap also recommends importing from the specific file like so, but the lack of auto-complete for this import style makes it tedious.
```
import Card from 'react-bootstrap/Card'
```

https://react-bootstrap.netlify.app/docs/getting-started/introduction#importing-components

Reply:
I mixed syntax from those two styles. Good to know which one to go with in any case.